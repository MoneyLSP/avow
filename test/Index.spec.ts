import Index from "@/pages/index.vue"
import Vuetify from "vuetify"
import { createLocalVue, mount, MountOptions } from "@vue/test-utils"
import Vue from "vue"

describe("Index Page", () => {
  const localVue = createLocalVue()
  let vuetify: any
  beforeEach(() => {
    vuetify = new Vuetify()
  })

  const mountFunction = (options: MountOptions<Vue>) => {
    return mount(Index, {
      localVue,
      vuetify,
      ...options,
    })
  }

  it("should mount properly", () => {
    const wrapper = mountFunction({})
    expect(wrapper.exists()).toBe(true)
  })
  it("should match snapshot", () => {
    const wrapper = mountFunction({})
    expect(wrapper.html()).toMatchSnapshot()
  })
})
