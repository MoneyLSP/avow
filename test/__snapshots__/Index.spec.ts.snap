// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Page should match snapshot 1`] = `
"<div class=\\"container\\">
  <div data-testid=\\"index-page\\" class=\\"row align-center justify-center\\">
    <div class=\\"col-sm-8 col-md-6 col-12\\"></div>
  </div>
</div>"
`;
